test/index.spec.js

```diff
@@ -1,53 +1,198 @@
 /* eslint-env mocha */
 import { expect } from 'chai'
 import * as lawReg from '../src/index.js'
-import * as log from 'loglevel'
+// import * as log from 'loglevel'
 
 import awb from './flint-example-awb'
 
 // import { loadConnector } from '../src/connector-loader.js'
 
 // Adjusting log level for debugging can be done here, or in specific tests that need more finegrained logging during development
-log.getLogger('disciplLawReg').setLevel('warn')
```

Comment:
I sort of left this here in order to instruct how to use the loglevel, so this is the only one that we would leave. I'm also okay with removing this, but then also remove the comment above it and the import.

Reply:
i added it